src/io/files.js

```diff
@@ -125,7 +125,7 @@ p5.prototype.loadFont = function(path, onSuccess, onError) {
     }
     /*jshint multistr: true */
     var exp =/\/[a-zA-Z]*((.ttf)|(.otf)|(.woff)|(.woff2))$/i;
```

Comment:
I'm wondering if this regex is too strict? Should it also contain hyphens and numeric characters, maybe? I'm too unfamiliar with the code to know for sure, I guess.